lib/api/i14y_search_options.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
Copy/pasted out of `lib/api/non_commercial_search_options.rb` and into own file.